### PR TITLE
improve usability of adding passengers

### PR DESCRIPTION
**Pull request by josephhirsch**

### CODE REVIEW

File: src/app/reimbursement-control.service.ts [hunk 1]
```diff
@@ -183,10 +191,8 @@ export class ReimbursementControlService {
 
   getExpense(value: ExpenseFormValue): Expense {
     if (value.type === 'car') {
-      const passengers = value.passengers
-        ? value.passengers.split(',').map(passenger => passenger.trim())
-        : [];
-      return { ...value, passengers } as Expense;
+      const passengers = value.passengers;
+      return { ...value, passengers: passengers ?? [] } as Expense;
```
Comment (simon-lgb) [inline src/app/reimbursement-control.service.ts:195]: This if block could be removed now.
This function is currently not type-safe as there is no way to tell typescript which form controls will be present based on the type input. We just rely on the controls returned by getExpenseFormGroup() not being modified at the moment. We could check that every value is defined by hand, but if we decide to do that I would start a seperate commit.

File: src/app/reimbursement-control.service.ts [hunk 2]
```diff
@@ -262,6 +272,7 @@ export class ReimbursementControlService {
     const outbound = this.getExpenses('outbound');
     outbound.clear();
 
+    console.log(values);
```
Comment (simon-lgb) [inline src/app/reimbursement-control.service.ts:275]: Sind die console logs absichtlich drin?

File: src/app/travel-expenses/expenses-collection/add-expense-modal/add-expense-modal.component.html [hunk 3]
```diff
@@ -147,30 +147,28 @@ <h3>{{ getExpenseName() }} hinzufügen</h3>
 
     <div *ngIf="passengers" class="form-group">
       <label for="passengers">Mitfahrer*innen</label>
-      <p>
-        Bitte gib die Namen der Mitfahrer*innen an. Z.B. "Kathi, Julia Fendt,
-        Peter Zimmermann", oder lass das Feld leer, wenn Du alleine gefahren
-        bist.
-      </p>
-      <p>
-        <b>Achtung:</b> Bist Du Teile der Strecke mit unterschiedlicher Anzahl
```
Comment (simon-lgb) [inline src/app/travel-expenses/expenses-collection/add-expense-modal/add-expense-modal.component.html:156]: Sollten wir diesen Hinweis drin lassen?

**Review by simon-lgb — APPROVED** [review]

File: src/app/reimbursement-control.service.ts (hunk 2 above)
Comment (josephhirsch) [reply src/app/reimbursement-control.service.ts:275]: Haha nein 🙈 Ups...

File: src/app/travel-expenses/expenses-collection/add-expense-modal/add-expense-modal.component.html (hunk 3 above)
Comment (josephhirsch) [reply src/app/travel-expenses/expenses-collection/add-expense-modal/add-expense-modal.component.html:156]: Ah ja klar.

Comment (josephhirsch) [reply src/app/travel-expenses/expenses-collection/add-expense-modal/add-expense-modal.component.html:156]: done

File: src/app/reimbursement-control.service.ts (hunk 2 above)
Comment (josephhirsch) [reply src/app/reimbursement-control.service.ts]: removed

File: src/app/reimbursement-control.service.ts (hunk 1 above)
Comment (josephhirsch) [reply src/app/reimbursement-control.service.ts]: removed

**Pull request merged**